CONTRIBUTING.md

```diff
@@ -177,4 +177,4 @@ To help keep CI green, please make sure of the following:
 
 ### Extra Credit
 
-- Our CI tests will finish early if you don't make changes to certain directories. If you want to **get results faster** then you should make changes to **docs** directory in one PR, and changes to anything else in another!
+- Our CI tests will finish early if you don't make changes to certain directories. If you want to **get results faster** then you should make changes to **docs** directory in one PR, and changes to anything else in another!
```

Comment:
not sure why this got marked as a change in git 🤔

Reply:
Make by due to space to tab conversion or vice versa?

Reply:
Maybe there's a Prettier formatting (although that seems unlikely here). No worries, on this, I'll check it out later.